CRM/Contact/BAO/Query.php

```diff
@@ -5676,6 +5676,9 @@ public static function buildClause($field, $op, $value = NULL, $dataType = NULL)
           $value = array($op => (array) $value);
         }
 
+      case 'RLIKE':
+        return " {$clause} BINARY '{$value}' ";
```

Comment:
@monishdeb test failures are related and seem to be hitting this line here complaining about an array to string conversion. I'm wondering if the change above with the caseImportant change is proving to be problematic